README.md

```diff
@@ -6,24 +6,40 @@ RNA-seq reporting workflow designed to post-process, summarise and visualise an
 ## Table of contents
 
 <!-- vim-markdown-toc GFM -->
-* [Installation](#installation)
-* [Docker](#docker)
-* [Workflow](#workflow)
-* [Reference data](#reference-data)
-    * [External reference cohorts](#external-reference-cohorts)
-    * [Internal reference cohort](#internal-reference-cohort)
-* [Input data](#input-data)
-  * [WTS](#wts)
-     * [bcbio-nextgen](#bcbio-nextgen)
-     * [Dragen RNA](#dragen-rna)
-  * [WGS](#wgs)
-* [Usage](#usage)
-  * [Arguments](#arguments)
-  * [Examples](#examples)
-  	 * [WTS data only](#1-wts-data-only)
-  	 * [WTS and WGS data](#2-wts-and-wgs-data)
-  	 * [WTS WGS and clinical data](#3-wts-wgs-and-clinical-data)
-  * [Output](#output)
+- [RNAsum](#rnasum)
+  - [Table of contents](#table-of-contents)
+  - [Installation](#installation)
+  - [Docker](#docker)
+  - [Workflow](#workflow)
+  - [Reference data](#reference-data)
+    - [External reference cohorts](#external-reference-cohorts)
+          - [Note](#note)
+    - [Internal reference cohort](#internal-reference-cohort)
+          - [Note](#note-1)
```

Comment:
as above, do we need to list "notes"?

Reply:
Updated in https://github.com/umccr/RNAsum/pull/46/commits/16dfd3d474ef85becc66ddcd67da1b44dd38f0da